### ISSUE TITLE
[FEATURE] KNX Support

### ISSUE THREAD

**Issue by qzwzqty** [thingsboard/thingsboard-gateway#936]
**Describe the issue**
Hope the gateway can support KNX protocol

**Comment by samson0v** [comment]
Hi @qzwzqty, Gateway doesn't support the KNX protocol.

**Comment by qzwzqty** [comment]
@samson0v Thank you for your reply, are there any plans to support the knx protocol?

**Comment by samson0v** [comment]
@qzwzqty, yes, we will add this protocol to Gateway but not in the nearest feature.
Meanwhile, I found a python library [XKNX](https://xknx.io/introduction.html) for implementation, take a look at it and let us know if this library meets your needs.

**Comment by qzwzqty** [comment]
@samson0v Thanks, this library is OK.
